history-of-quakers/modernized/06-chapter-4.adoc

```diff
@@ -449,8 +484,13 @@ though F. Howgill and E. Burrough had been required to do so.
 So this warrant proving ineffectual,
 the magistrates caused another to be formed in these words:
 
+[.embedded-content-document.legal]
+--
+
+[.signed-section-context-open]
 City of Bristol.
 
+[.letter-heading]
```

Comment:
same here, let's make it a 'salutation', since it's _below_ the context-open above.